checkov/common/util/docs_generator.py

```diff
@@ -46,20 +46,21 @@ def get_compare_key(c: list[str] | tuple[str, ...]) -> list[tuple[str, str, int,
 
 
 def print_checks(frameworks: Optional[List[str]] = None, use_bc_ids: bool = False,
-                 include_all_checkov_policies: bool = True) -> None:
+                 include_all_checkov_policies: bool = True, filtered_policy_ids: List[str] = []) -> None:
```

Comment:
please don't use `[]` as a default value, because it mutable, you can use `None` and then check, if it is `None`, then set `[]`. Using a mutable object as a default in Python results in surprising behaviour, because it is reused with consecutive calls.

Reply:
Got it! Thanks for pointing this out